lib/auth_mvp_web/controllers/github_auth_controller.ex

```diff
@@ -1,23 +1,35 @@
 defmodule AuthMvpWeb.GithubAuthController do
   use AuthMvpWeb, :controller
-  
+
   @doc """
   `index/2` handles the callback from GitHub Auth API redirect.
   """
   def index(conn, %{"code" => code, "state" => client}) do
     {:ok, profile} = ElixirAuthGithub.github_auth(code)
 
-    {:ok, session} = case AuthMvp.People.get_person_by_email(profile.email) do
-      nil ->
-        {:ok, person} = AuthMvp.People.create_person(%{email: profile.email})
-        AuthMvp.Email.sendemail(%{email: profile.email, template: "welcome"})
-        AuthMvp.People.create_session(person)
+    {:ok, session} =
+      case AuthMvp.People.get_person_by_email(profile.email) do
+        nil ->
+          {:ok, person} =
+            AuthMvp.People.create_person(%{email: profile.email, avatar: profile.avatar_url})
+
+          # AuthMvp.Email.sendemail(%{email: profile.email, template: "welcome"})
+          AuthMvp.People.create_session(person)
+
+        person ->
+          {:ok, p} = AuthMvp.People.update_person(person, %{avatar: profile.avatar_url})
+          AuthMvp.People.create_session(p)
+      end
+
+    person = AuthMvp.People.get_person_by_email(profile.email)
```

Comment:
Make sure to get the latest version of the person before creating the jwt based on this information